doc/PYTHON_ROUTING.md

```diff
@@ -0,0 +1,24 @@
+# Python Routing
+
+* [Summary](#summary)
+
+## Summary
+
+This describes how to use the Python-based t-route routing module with ngen. 
+
+The basic outline of steps needed to run the Remote Nexus Class with MPI is:
+  * [Set up pybind11] (https://github.com/robertbartel/ngen/blob/python/main/doc/DEPENDENCIES.md#pybind11)
+    UPDATE ABOVE LINK WHEN MERGED INTO MASTER
+ 
+  * Create the build directory including the options to activate Python and Routing: 
+  
+      `cmake -DCMAKE_BUILD_TYPE=Debug -B cmake-build-debug -DNGEN_ACTIVATE_PYTHON:BOOL=ON -DNGEN_ACTIVATE_ROUTING:BOOL=ON -DBMI_C_LIB_ACTIVE:BOOL=ON -S .`  
```

Comment:
It might be better to note the specific options that need to be included.  Then, have the example, but point out the caveat that some of the other flags in have their own use and may or may not be needed.   This just seems like it could be copied without complete understanding/awareness of the other items (e.g., the BMI C activation), and then have things turned on unintentionally.

Might also be good to link back to the document on regenerating the CMake build directory.

Reply:
Good point. Will do.